tests/test_project.py

```diff
@@ -70,6 +71,63 @@ def test_logging_experiment(project_kwargs):
         project["not a real experiment"]
 
 
+LOG = logging.getLogger(__name__)
+
+
+@time_machine.travel(
+    datetime(2025, 1, 20, 13, 23, 30, tzinfo=zoneinfo.ZoneInfo("UTC")), tick=False
+)
+@pytest.mark.parametrize(
+    "project_kwargs",
+    [
+        {"author": "root"},
+        {
+            "author": "root",
+            "fpath": "file://" + (DATA_DIR / "external_fs_project.json").as_posix(),
+        },
+    ],
+)
+def test_logging(caplog, project_kwargs):

```

Comment:
@gladysteh99 , nice to meet you!

There are already was the `test_logging_experiment`.
Is a separate test intended?

If so, should it be moved up as more common?